ktoml-source/src/commonMain/kotlin/com/akuleshov7/ktoml/source/SourceUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.akuleshov7.ktoml.source
+
+import okio.Source
+import okio.buffer
+import okio.use
+
+/**
+ * Read from source one line at a time and passes the lines to the [decoder] function.
+ */
+public inline fun <T> Source.useLines(decoder: (Sequence<String>) -> T): T {
```

Comment:
is it possible to rename it to something more clear? :)
`readAndDecodeLines` or something like this?